kie-server-parent/kie-server-tests/kie-server-integ-tests-common/src/main/java/org/kie/server/integrationtests/config/TestConfig.java

```diff
@@ -194,6 +194,15 @@ public static Integer getKieServerAllocatedPort() {
         return ALLOCATED_PORT;
     }
 
+    /**
+     * Reset all allocated ports so they can be dynamically reallocated again, to avoid BindException: Address already in use (Bind failed).
+     */
+    public static void resetAllocatedPorts() {
+        ALLOCATED_PORT = null;
+        CONTROLLER_ALLOCATED_PORT = null;
+        ROUTER_ALLOCATED_PORT = null;
```

Comment:
This doesn't solve the issue, right? It avoids it and at the same time is making it worse for those who come after.
Shouldn't we rather make sure we actually close those ports?